modules/programs/texlive.nix

```
@@ -14,21 +14,55 @@ in {
 
   options = {
     programs.texlive = {
-      enable = mkEnableOption "TeX Live";
+      enable = mkEnableOption ''
+        TeX Live package customization.
+
+        This module allows you to select which `pkgs.texlivePackages.*` you want to install.
+        Start by editing {option}`programs.texlive.extraPackages`.
+
+        If one of the pre-built common environments already suits you
+        (e.g. `pkgs.texliveFull` or `pkgs.texliveMedium`),
+        you do not need to use this module.
+      '';
```

The `mkEnableOption` function automatically appends a `.` to this string so you can omit that (and the newline character to avoid a space between module and `.`: "module ."

```suggestion
        you do not need to use this module'';
```